src/index.js

```diff
@@ -1,17 +1,13 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './core/App';
-import reportWebVitals from './reportWebVitals';
-import { GlobalStyle } from './GlobalStyle';
-import { theme } from './theme'
-import { ThemeProvider } from 'styled-components';
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./core/App";
+import reportWebVitals from "./reportWebVitals";
+import { ThemeProvider } from "styled-components";
```

Comment:
czy to nie problem, albo zbędna rzecz, że ThemeProvider dubluje się tutaj i w App.js?